### ISSUE TITLE
Code Syntax highlighting is messed up all of a sudden.

### ISSUE THREAD

**Issue by omarcevi** [colineckert/obsidian-things#147]
Hello, 

After updating to the last version of the theme, my code blocks syntax highlighting is all messed up and showing mainly 1 colour.
<img width="645" alt="Screenshot 2023-09-02 at 20 51 29" src="https://github.com/colineckert/obsidian-things/assets/63335393/5114d743-d539-4072-9ab1-d6a2ef2b3258">

anybody facing the same issue?

Edit:
This was the syntax before i updated
<img width="451" alt="Screenshot 2023-09-02 at 20 59 44" src="https://github.com/colineckert/obsidian-things/assets/63335393/5ece63e5-587e-4810-955e-ac6a8ebb1088">

**Comment by omarcevi** [comment]
Yeah so i manually reverted to an older version (2.0.0) and the syntax was back normal.

**Comment by colineckert** [comment]
hey @lukasschwab and @omarcevi, thanks for creating the issue. i just released a new version of the theme that should address some of your concerns with syntax highlighting: https://github.com/colineckert/obsidian-things/releases/tag/2.1.15

lmk if this looks better and if so, pls close this issue. thanks!

**Comment by lukasschwab** [comment]
Hi @colineckert — this is my first time seeing this thread. Was there something you wanted me to look at?

**Comment by colineckert** [comment]
oops sorry @lukasschwab. you're good, i `@` you by accident.

**Comment by omarcevi** [comment]
Yeah its fixed now closing the ticket :)

**Comment by omarcevi** [comment]
Hello again, the code blocks are all fine in the editing view, but they're not in the reading view. It affects the publish feature. the syntax coloring is not the same.

**Comment by svenblum** [comment]
I can confirm the issue. Editing is fine, viewing is single-color (red) only. Would be great to have this fixed.

Edit: I just realized that it's depending on the language. So it's only half-broken :-) See attached screenshots
Edit2: I think it takes two different sources for the coloring. The "-m" which has a darker brown in edit-mode is not colored differently in bash.
Edit3: My iterm2 with https://github.com/zsh-users/zsh-syntax-highlighting renders different colors. Maybee it could be nice to select the desired "color-scheeme"? but I think I'm asking too much here...

![CleanShot 2024-04-10 at 08 15 58@2x](https://github.com/colineckert/obsidian-things/assets/14309900/d1803da8-6f11-4c15-9eb0-adb1dc0c85b3)
![CleanShot 2024-04-10 at 08 16 12@2x](https://github.com/colineckert/obsidian-things/assets/14309900/cf2feeb8-1918-45a3-96f2-1eed09608287)
![CleanShot 2024-04-10 at 08 34 39@2x](https://github.com/colineckert/obsidian-things/assets/14309900/fd3505a3-cd89-40e1-966a-8c831170ef5f)
